spec/spec_join_addfields/section/join-vocabulary.md

```diff
@@ -0,0 +1,116 @@
+## Join vocabulary {#join-vocabulary}
+
+The Join vocabulary namespace is http://w3id.org/rml/
+and it's prefix is `rml`.
+
+### Joins
+
+A <dfn>Join</dfn> (`rml:Join`) is an operation that extends one logical source (the child logical source) with data from another logical source (the parent logical source).
+
+A [=Join=] (`rml:Join`) MUST contain:
+- exactly one [=parent logical source=] property (`rml:parentLogicalSource`) to describe the data source that supplies the additional data. 
+- at least one [=join condition=] property (`rml:joinCondition`) to describe which values are compared to join the two data sources.
+- at least one [=add field=] property (`rml:addField`) or one [=add field alias=] property (`rml:addFielAlias`) to describe a field from the parent logical source that is added to the child logical source. 
+
+| Property                  | Domain     | Range               |
+|---------------------------|------------|---------------------|
+| `rml:parentLogicalSource` | `rml:Join` | `rml:LogicalSource` |
+| `rml:joinCondition`       | `rml:Join` | `rml:JoinCondition` |
+| `rml:addField`            | `rml:Join` | `Literal`           |
+| `rml:addFieldAlias`       | `rml:Join` | `rml:FieldAlias`    | 
+
+<aside class="issue">
+Els: in the RML CORE spec rml:Join is the class containing the join conditions. Can this still be adapted?
+</aside>
+
+### Parent Logical Source
+
+The Range of a <dfn>parent logical source</dfn> property is a [Logical Source](https://kg-construct.github.io/rml-io/spec/docs/#defining-logical-sources) and describes the data source that supplies the additional data and is the right side of the join. 
+
+### Join Conditions
```

Comment:
TODO: clarify that this is in fact exactly the same construct/definition as in RML-Core? Or are there differences?